### PR TITLE
Site uniq by location

### PR DESCRIPTION
**Pull request by carlosmartinez**
Trello: https://trello.com/c/I3TjV5oi/292-site-uuids-provider-details-average-distance-to-site

### CODE REVIEW

File: app/models/temp_site.rb [hunk 1]
```diff
@@ -18,6 +18,10 @@ def self.for_recruitment_cycle_years(recruitment_cycle_years)
     .distinct
   end
 
+  def self.uniq_by_location
+    all.uniq { |site| site.slice(:longitude, :latitude) }
```
Comment (tomas-stefano) [inline app/models/temp_site.rb:22]: Is it a problem to load all records into memory?

Comment (tomas-stefano) [reply app/models/temp_site.rb:22]: If I understood correctly the `format_average_distance` only calls `#longitude` and `#latitude` on the object passed in the view.

Based on that can we do something like this?

```
  all.select(:longitude, :latitude).uniq
```

But happy as it is. This is minor thing.

**Review by tomas-stefano — APPROVED** [review]
Minor questions but all looks good! 🚀  (after the build passes)

File: app/models/temp_site.rb (hunk 1 above)
Comment (carlosmartinez) [reply app/models/temp_site.rb]: Thanks @tomas-stefano. TIL Rails doesn't support `DISTINCT ON`, so have just used SQL. I think this addresses the performance issue.

**Pull request merged**